misc/docs/source/packages/obspy.io.nied.rst

```diff
@@ -0,0 +1,14 @@
+.. currentmodule:: obspy.io.nied
+.. automodule:: obspy.io.nied
+
+    .. comment to end block
+
+    Modules
+    -------
+    .. autosummary::
+       :toctree: autogen
+       :nosignatures:
+
+       core
```

Comment:
There is no `core` within `obspy.io.nied`. In fact, I think you need to list `fnetmt` and `knet` here and drop those other two files.